Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:10
+WORKDIR /app
+COPY package.json .
+RUN npm install
+COPY . .
```

Comment:
Do you need to bulid the bundle.js files?  Or do you have then pre-built and then just copy them over?